### ISSUE TITLE
[BUG] ImportError: cannot import name '_fetch_files' from 'nilearn.datasets.utils'

### ISSUE THREAD

**Issue by Vincent-wq** [netneurolab/neuromaps#176]
### Issue summary

Looks like an import error from nilearn update, I feel like it may be related to the fetcher description update. What do you think? 
[nilearn update](https://nilearn.github.io/stable/changes/whats_new.html)

### Detailed issue description

Code: `import nilearn
print('nilearn_version ', nilearn.__version__)
import neuromaps 
print('neuromaps_version ', neuromaps.__version__)`
Output: `nilearn_version  0.10.4`
Error: 
```
Cell In[5], [line 3](vscode-notebook-cell:?execution_count=5&line=3)
      [1](vscode-notebook-cell:?execution_count=5&line=1) import nilearn
      [2](vscode-notebook-cell:?execution_count=5&line=2) print('nilearn_version ', nilearn.__version__)
----> [3](vscode-notebook-cell:?execution_count=5&line=3) import neuromaps 
      [4](vscode-notebook-cell:?execution_count=5&line=4) print('neuromaps_version ', neuromaps.__version__)
 
File /opt/conda/lib/python3.9/site-packages/neuromaps/__init__.py:3
      [1](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/__init__.py:1) __all__ = ['resample_images', 'compare_images']
----> [3](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/__init__.py:3) from neuromaps.resampling import resample_images
      [4](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/__init__.py:4) from neuromaps.stats import compare_images
      [6](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/__init__.py:6) from ._version import get_versions

File /opt/conda/lib/python3.9/site-packages/neuromaps/resampling.py:9
      [6](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/resampling.py:6) import nibabel as nib
      [7](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/resampling.py:7) import numpy as np
----> [9](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/resampling.py:9) from neuromaps import transforms
     [10](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/resampling.py:10) from neuromaps.datasets import ALIAS, DENSITIES
     [11](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/resampling.py:11) from neuromaps.images import load_gifti, load_nifti

File /opt/conda/lib/python3.9/site-packages/neuromaps/transforms.py:14
     [11](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/transforms.py:11) import numpy as np
     [12](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/transforms.py:12) from scipy.interpolate import interpn
---> [14](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/transforms.py:14) from neuromaps.datasets import (ALIAS, DENSITIES, fetch_atlas,
     [15](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/transforms.py:15)                                 fetch_regfusion, get_atlas_dir)
     [16](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/transforms.py:16) from neuromaps.images import (construct_shape_gii, load_gifti, load_nifti,
     [17](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/transforms.py:17)                               load_data)
     [18](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/transforms.py:18) from neuromaps.utils import tmpname, run

File /opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:17
     [13](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:13) import warnings
     [14](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:14) warnings.filterwarnings('ignore', message='Fetchers from the nilearn.datasets',
     [15](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:15)                         category=FutureWarning)
---> [17](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:17) from .atlases import (fetch_all_atlases, fetch_atlas, fetch_civet,  # noqa
     [18](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:18)                       fetch_fsaverage, fetch_fslr, fetch_mni152,
     [19](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:19)                       fetch_regfusion, get_atlas_dir, DENSITIES, ALIAS)
     [20](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:20) from .annotations import (available_annotations, available_tags,  # noqa
     [21](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:21)                           fetch_annotation)
     [22](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/__init__.py:22) from .contributions import (upload_annotation)

File /opt/conda/lib/python3.9/site-packages/neuromaps/datasets/atlases.py:10
      [7](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/atlases.py:7) import os
      [8](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/atlases.py:8) from pathlib import Path
---> [10](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/atlases.py:10) from nilearn.datasets.utils import _fetch_files
     [11](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/atlases.py:11) from sklearn.utils import Bunch
     [13](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f616e6772795f6275726e656c6c227d.vscode-resource.vscode-cdn.net/opt/conda/lib/python3.9/site-packages/neuromaps/datasets/atlases.py:13) from neuromaps.datasets.utils import get_data_dir, get_dataset_info

ImportError: cannot import name '_fetch_files' from 'nilearn.datasets.utils' (/opt/conda/lib/python3.9/site-packages/nilearn/datasets/utils.py)
'''

### Steps to reproduce issue

```python
>>>import nilearn
>>>print('nilearn_version ', nilearn.__version__)
>>>import neuromaps 
>>>print('neuromaps_version ', neuromaps.__version__)
```

### Software version

nilearn_version  0.10.4
neuromaps_version: 0.0.5

### Code of Conduct

- [X] I agree to follow the `neuromaps` Code of Conduct

**Comment by liuzhenqi77** [comment]
Hi thanks for the issue! 

This should have been fixed in #146 and released with neuromaps 0.0.5.
Could you try updating neuromaps to the latest PyPI version (0.0.5) or to GitHub main branch and try again please?

Zhen-Qi

**Comment by Vincent-wq** [comment]
Hi, thanks! I just realized that my previous installation version is `0+untagged.223.g539d80e`, I was able to update neuromaps to 0.0.5 with `pip install neuromaps==0.0.5`. Thanks!
Qing

**Comment by Vincent-wq** [comment]
Hi Zhen-Qi,
I just tried to download a neuromap and play around with it, but neuromaps is complaining about a missing function `neuromaps.datasets.describe_annotations`. I tried the github installation then, and it works perfectly well. My current version is `0.0.5+27.ga89b699`.
Cheers,
Qing

**Comment by liuzhenqi77** [comment]
Thanks for the feedback! Indeed `describe_annotations()` was added a few days ago and was not shipped with 0.0.5. However we'll include it in the next release hopefully happening soon. Please feel free to let us know if you meet any further issues.

Zhen-Qi
